src/Card/CardImageCap.jsx

```diff
@@ -45,26 +46,37 @@ const CardImageCap = React.forwardRef(({
     );
   }
 
-  const handleSrcFallback = (event, altSrc) => {
+  const handleSrcFallback = (event, altSrc, imageKey) => {
     const { currentTarget } = event;
-    if (currentTarget.src !== altSrc) {
-      currentTarget.src = altSrc;
+
+    if (!altSrc || currentTarget.src.endsWith(altSrc)) {
```

Comment:
nit/curious: could this be simplified to `if (!altSrc || currentTarget.src === altSrc) {`?

Reply:
Using the option you suggested, we run the risk of getting an infinite image `src` call. This comparison `!altSrc || currentTarget.src === altSrc` successfully processes the passed link, but if we try to pass a string value to `fallbackLogoSrc` (for example, `fallbackLogoSrc="undefined "`), we will get an endless request along the path passed to the `src`.

![image](https://user-images.githubusercontent.com/93188219/217665869-d0c74551-2a5c-4191-b17b-2cb1ed9828b7.png)

Reply:
yeah, doing just `currentTarget.src === altSrc` fails if you provide relative url to the image instead of full url

Reply:
Thanks for the clarification 👍